css-view-transitions-2/Overview.bs

```diff
@@ -414,3 +411,25 @@ The <dfn attribute for=PageRevealEvent>viewTransition</dfn> [=getter steps=] are
 This specification introduces no new privacy considerations.
 
 <h2 id="sec" class="no-num">Security Considerations</h2>
+
+To prevent cross-origin issues, at this point cross-document view transitions can only be enabled for
+same-origin navigations. As discussed in <a href="https://github.com/WICG/view-transitions/issues/200">WICG/view-transitions#200</a>,
+this still presents two potential threats:
+
+1. The <a data-xref-type="http-header">Cross-Origin-Opener-Policy</a> of both documents might be different.
+	This can cause a situation where a {{Document}} that is [=environment settings object/cross-origin isolated capability|cross-origin isolated=]
+	can read image data from a document that is not cross-origin isolated. To mitigate this, implementations should
+	keep the image data in an isolated memory space (e.g. a separate process), not directly accessible to the new {{Document}}.
```

Comment:
This requirement is already there because of cross-origin iframes being a part of captured images: https://drafts.csswg.org/css-view-transitions-1/#sec. So we can point to that to argue why we're not veto-ing transitions between Documents with different COOP policies.